tests/regression_tests/kotahi_docmaps/kotahi_docmap_manuscript_test.py

```diff
@@ -0,0 +1,74 @@
+import os
+import pytest
+import requests
+import json
+
+DEFAULT_DATA_HUB_API_REGRESSION_TEST_URL_PREFIX = 'http://localhost:8000'
+DATA_HUB_API_REGRESSION_TEST_URL_PREFIX_ENV = 'DATA_HUB_API_REGRESSION_TEST_URL_PREFIX'
+
+DOCMAP_BY_MANUSCRIPT_PATH = (
+    '/kotahi/docmaps/v1/by-publisher/elife/get-by-manuscript-id'
+)
+
+MANUSCRIPT_ID_LIST = [
+    '93934'  # evaluations for 2 versions of manuscript
+]
+
+NOT_AVAILABLE_DOCMAP_JSON = {
+  "detail": "No Docmaps available for requested manuscript from the publisher eLife"
+}
+
+
+@pytest.fixture(name='regression_test_url_prefix')
+def _regression_test_url_prefix() -> str:
+    return os.getenv(
+        DATA_HUB_API_REGRESSION_TEST_URL_PREFIX_ENV,
+        DEFAULT_DATA_HUB_API_REGRESSION_TEST_URL_PREFIX
+    )
+
+
+@pytest.fixture(name='regression_test_docmap_by_manuscript_url')
+def _regression_test_docmap_by_manuscript_url(
+    regression_test_url_prefix: str
+) -> str:
+    return regression_test_url_prefix + DOCMAP_BY_MANUSCRIPT_PATH
+
+
+#  qc_complete_timestamp > '2023-10-01' for each manuscript version
```

Comment:
Would suggest to rename the test to reflect the intention (instead of date, could also refer to cutoff or similar date). e.g. something like:
`test_docmaps_should_not_be_available_for_manuscript_first_submitted_before_cutoff_date`